README.md

```diff
@@ -134,6 +134,23 @@ The followup screen can be skipped for promoters, so they are taken straight to
 wootric.shouldSkipFollowupScreenForPromoters(true);
 ```
 
+### Color customization (smartphones only)
+Colors can be customized for the smartphone's version.
+
+```java
+// Changes background color and text buttons color for the survey
+wootric.setSurveyColor(R.color.survey_color);
```

Comment:
I just found the `colors.xml` file you should indicate that the values come from there